rasa/utils/train_utils.py

```diff
@@ -174,23 +172,35 @@ def update_evaluation_parameters(config: Dict[Text, Any]) -> Dict[Text, Any]:
 
 
 def load_tf_hub_model(model_url: Text) -> Any:
-    """Load model from cache if possible, otherwise from TFHub"""
+    """Load model from cache if possible, otherwise from TFHub."""
+    import os
+
+    # Once `tensorflow-hub` have rebuilt their `_pb2.py` files with protobuf-4.x
+    # this can be removed.
+    if os.getenv("PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION", "ubp") != "python":
```

Comment:
PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION should be a constant in related constants.py.